codespell_lib/data/dictionary.txt

```diff
@@ -14438,10 +14476,18 @@ copmutations->computations
 copntroller->controller
 coponent->component
 coponents->components
+coporate->corporate
+coporation->corporation
+coporations->corporations
 copoying->copying
 coppermines->coppermine
 coppied->copied
+coppies->copies
```

Comment:
Perhaps `coppices`, and `coppiced` in the line before?

Reply:
This is rare, but exists. I'll add these.